src/main/resources/static/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    E-Commerce!
+</body>
+</html>
```

Comment:
<img width="173" height="58" alt="Screenshot 2025-07-19 at 10 03 27 PM" src="https://github.com/user-attachments/assets/f03832ec-2e54-4713-9c74-ee0b23ee009f" />

Let's not forget to add an empty line at the end of all the files we try to merge.
Please check other files in the project as well.